### PR TITLE
Added Midi event trigger for panic button and File/Exit menu

### PR DESCRIPTION
**Pull request by lupushortus**
Resolves: #1807
Resolves: #1905
Setter class has two new virtual buttons, that provide MIDI access to panic button and exit function
Implements Issues #1905 and #1807

### CODE REVIEW

File: src/grandorgue/combinations/GOSetter.cpp [hunk 1]
```diff
@@ -330,6 +333,8 @@ GOSetter::GOSetter(GOOrganController *organController)
   m_buttons[ID_SETTER_TEMPERAMENT_NEXT]->SetPreconfigIndex(22);
   m_buttons[ID_SETTER_TRANSPOSE_DOWN]->SetPreconfigIndex(23);
   m_buttons[ID_SETTER_TRANSPOSE_UP]->SetPreconfigIndex(24);
+  m_buttons[ID_SETTER_AUDIO_PANIC]->SetPreconfigIndex(25);
+  m_buttons[ID_SETTER_FILE_EXIT]->SetPreconfigIndex(25);
```
Comment (oleg68) [inline src/grandorgue/combinations/GOSetter.cpp:337]:
```suggestion
  m_buttons[ID_SETTER_FILE_EXIT]->SetPreconfigIndex(26);
```

**Review by oleg68 — CHANGES_REQUESTED** [review]

**Comment by oleg68** [comment]
@lupushortus 

1. Please remove any changes from the `submodules/RtAudio` submodule. They do not relate to this functionality.
2. How a user can assign MIDI settings to these hidden buttons?

**Comment by lupushortus** [comment]
Hi @oleg68,

1.  I struggled merging/updating upstream branch into my fork/branch handling the submodule RtAudio correctly. After some forced pushes I integrated it cleanly. Sorry for the confusion.

2. Regarding the hidden buttons: Go to menu Audio/Midi => MIDI Objects, there's a list of all MIDI elements available, search and select button Panic or Exit GrandOrgue, that appear with that change and press Configure button. Standard MIDI config dialogue will appear.

**Review by oleg68 — APPROVED** [review]

**Review by oleg68 — CHANGES_REQUESTED** [review]
@lupushortus Could you add a changelog entry and a sescriprion of new setter elements to the help file?

**Comment by lupushortus** [comment]
> @lupushortus Could you add a changelog entry and a sescriprion of new setter elements to the help file?

Done

**Comment by oleg68** [comment]
@lupushortus please add also the descriprion of Exit and of the new setter elements in the `Setter elements` section

**Review by oleg68 — CHANGES_REQUESTED** [review]
@lupushortus please add also the descriprion of Exit and of the new setter elements in the Setter elements section

**Comment by lupushortus** [comment]
> @lupushortus please add also the descriprion of Exit and of the new setter elements in the Setter elements section

Done, also added links to relevant items.

**Review by oleg68 — APPROVED** [review]

**Review by larspalo — APPROVED** [review]

**Pull request merged**